website/src/parsers/scala/index.js

```diff
@@ -0,0 +1,6 @@
+import 'codemirror/mode/clike/clike';
+
+export const id = 'text/x-scala';
```

Comment:
I had to use this in order to get the correct syntax highlight in codemirror. Is there any other way to set-up codemirror other than the category id?